clients/src/main/java/org/apache/kafka/clients/consumer/OffsetAndTimestamp.java

```diff
@@ -45,7 +43,7 @@ public String toString() {
 
     @Override
     public int hashCode() {
-        return Objects.hash(timestamp, offset);
+        return 31 * Long.valueOf(timestamp).hashCode() + Long.valueOf(offset).hashCode();
```

Comment:
Also, can we add javadoc for `offset` and `timestamp`? This is a public class so it's good to add some documentation.